scripts/setup-helper-functions.sh

```diff
@@ -58,81 +58,59 @@ function github_checkout {
   cd "${DIRNAME}"
 }
 
-# get_cxx_flags [$CPU_ARCH]
-# Sets and exports the variable VELOX_CXX_FLAGS with appropriate compiler flags.
-# If $CPU_ARCH is set then we use that else we determine best possible set of flags
-# to use based on current cpu architecture.
+# get_cxx_flags
 # The goal of this function is to consolidate all architecture specific flags to one
 # location.
+# Sets and exports the variable VELOX_CXX_FLAGS with appropriate compiler flags.
+# We determine best possible set of flags to use based on the cpu architecture CPU_ARCH.
 # The values that CPU_ARCH can take are as follows:
 #   arm64  : Target Apple silicon.
 #   aarch64: Target general 64 bit arm cpus.
 #   avx:     Target Intel CPUs with AVX.
 #   sse:     Target Intel CPUs with sse.
 # Echo's the appropriate compiler flags which can be captured as so
-# CXX_FLAGS=$(get_cxx_flags) or
-# CXX_FLAGS=$(get_cxx_flags "avx")
+# CXX_FLAGS=$(get_cxx_flags)
 
 function get_cxx_flags {
-  local CPU_ARCH=$1
-
-  local OS
-  OS=$(uname)
-  local MACHINE
-  MACHINE=$(uname -m)
-  ADDITIONAL_FLAGS=""
+  local CPU_ARCH="avx"
+  local OS=$(uname)
+  local MACHINE=$(uname -m)
 
-  if [[ -z "$CPU_ARCH" ]] || [[ $CPU_ARCH == "unknown" ]]; then
-    if [ "$OS" = "Darwin" ]; then
-
-      if [ "$MACHINE" = "x86_64" ]; then
-        local CPU_CAPABILITIES
-        CPU_CAPABILITIES=$(sysctl -a | grep machdep.cpu.features | awk '{print tolower($0)}')
-
-        if [[ $CPU_CAPABILITIES =~ "avx" ]]; then
+  if [ "$OS" = "Darwin" ]; then
+    if [ "$MACHINE" = "arm64" ]; then
+      CPU_ARCH="arm64"
```

Comment:
I suppose that mac's with cpu's that dont support avx are not supported anymore ?

Reply:
Sounds fair - just wanted to confirm.